src/test/java/burrows/apps/example/template/util/AdUtilsTest.java

```diff
@@ -19,6 +19,13 @@
 public class AdUtilsTest {
 
     @Test
+    public void test_constructor() {
+        // since there is only 1 static method in the class
```

Comment:
Yea, I observed that, I've tried to use mockito to instantiate those classes, and they were ok, and then removed them as your decision. Thanks!